src/Test/config.jl

```diff
@@ -9,9 +9,10 @@ struct TestConfig
     # The expected "optimal" status returned by the solver. Either
     # MOI.OPTIMAL or MOI.LOCALLY_SOLVED.
     optimal_status::MOI.TerminationStatusCode
+    basis::Bool # can get variable and constraint basis status
     function TestConfig(;atol::Float64 = 1e-8, rtol::Float64 = 1e-8, solve::Bool = true, query::Bool = true,
                         modify_lhs::Bool = true, duals::Bool = true, infeas_certificates::Bool = true,
-                        optimal_status = MOI.OPTIMAL)
+                        optimal_status = MOI.OPTIMAL, basis::Bool = false)
```

Comment:
`false` is the correct default value to avoid breaking all the solvers that run tests.